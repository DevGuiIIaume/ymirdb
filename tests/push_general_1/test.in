set a 1
set b a
set c 3
push c a
list entries
forward a
forward b
forward c
backward a
backward b
backward c
bye
