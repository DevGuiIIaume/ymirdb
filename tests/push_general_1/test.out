> ok

> ok

> ok

> ok

> c [a 3]
b [a]
a [1]

> nil

> a

> a

> b, c

> nil

> nil

> bye
